run.py

```diff
@@ -1,11 +1,23 @@
+#!/usr/bin/env python
+"""
+Run Covid data model.
+Execution:
+    ./run.py run-model [JHU|CDS] [--output-dir]
```

Comment:
1. Why do I need `run-model`?  What else does this script do?  Ideally it would do only one thing, I think. :-)
2. Why is it `JHU` and `CDS` instead of `--jhu` and `--cds` ?  I don't care super strongly, but I'm not sure if it makes sense.  FWIW we may end up with multiple model implementations, more datasets to choose from, and end up writing code to merge/blend them, etc.  So I don't know exactly how this will evolve.
3. From a documentation perspective, should this be `[--output-dir path]` to make it clear you have to pass an additional arg after `--output-dir`?

Reply:
It looks like there's a state parameter too, but you haven't documented that here?

Reply:
1. Down to remove that! I think i was intending on providing a single entry-point for various commands.  Other examples would be `.run.py validate` - validates data.  `./run.py download-local-data`.  
2. Generally the flags there would be additive.  For instance right now it wouldn't make sense to have `./run.py run-model --jhu --cds`.  Another solution would be `./run.py run-model --dataset [jhu|cds]`
3. yup!

It may not be good to keep the example here, i've found that it tends to get out of date quite quickly.  Running `./run.py run-model --help` shows the various command options.